### PR TITLE
Updates BYD sidebar Jul 18

### PR DESCRIPTION
**Pull request by dkluever22**
* Adds endpoints related to multilanguage in product-view (see https://github.com/ePages-de/ng-product-view/commit/4bb347d1)

### CODE REVIEW

File: _data/beyond-sidebar.yml [hunk 1]
```diff
@@ -1153,6 +1153,40 @@
...
+              id: show_product_details_for_a_given_locale
+            - title: List product cross-sells for a given locale
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-cross-sells-list-with-locale
+              id: list_product_crosssells_for_a_given_locale
+            - title: List cross-selling products for a given locale
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-list-cross-sell-products-with-locale
+              id: list_crossselling_products_for_a_given_locale
+        - title: Translated variation products
+          entries:
+            - title: Show variation product details for a given locale
```
Comment (AliceStatler) [inline _data/beyond-sidebar.yml:1184]: Same here

```suggestion
            - title: Show variation details for a given locale
```

File: _data/beyond-sidebar.yml [hunk 2]
```diff
@@ -1153,6 +1153,40 @@
...
+            - title: Show product details for a given locale
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-find-by-id-with-locale
+              id: show_product_details_for_a_given_locale
+            - title: List product cross-sells for a given locale
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-cross-sells-list-with-locale
+              id: list_product_crosssells_for_a_given_locale
+            - title: List cross-selling products for a given locale
+              link: https://api-docs.beyondshop.cloud/ng-product-view-product-view-public-api.html#product-view-list-cross-sell-products-with-locale
+              id: list_crossselling_products_for_a_given_locale
+        - title: Translated variation products
```
Comment (AliceStatler) [inline _data/beyond-sidebar.yml:1182]: Shouldn't this be "Translated variations" ? 

```suggestion
        - title: Translated variations
```

**Review by AliceStatler — APPROVED** [review]
@dkluever22 There you go :)

File: _data/beyond-sidebar.yml (hunk 2 above)
Comment (dkluever22) [reply _data/beyond-sidebar.yml:1182]: Nope! This refers to other headlines :)
Bildschirmfoto 2024-07-18 um 14 35 44
Bildschirmfoto 2024-07-18 um 14 31 51

File: _data/beyond-sidebar.yml (hunk 1 above)
Comment (dkluever22) [reply _data/beyond-sidebar.yml:1184]: Also not. The "variation details" endpoint is the next one and yes, this wording is also something that will be tackled in the second step, to reduce confusion.
Bildschirmfoto 2024-07-18 um 14 41 30

**Pull request merged**